Library/Homebrew/test/formula_info_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require "formula_info"
+require "global"
+
+describe FormulaInfo, :integration_test do
```

Comment:
Can these not be done without an `integration_test`? `integration_test`s are really slow because they reinitialise Homebrew every time. If not: please strip this down to a single test (that could have multiple assertions on the `lookup` return value instead)